SolarSharp.Interpreter.Tests/Units/AdversarialSecurityTests.cs

```diff
@@ -1343,106 +1343,6 @@ public void TestGarbageCollectionBypass()
             );
         }
 
-        /// <summary>
```

Comment:
This test just verifies the **code** has certain functions, this isn't really a test.  For one it hard codes paths, and for two it is pretty easy to avoid, tons of code changes that could "break" this won't trip this test.